README.adoc

```diff
@@ -5,3 +5,66 @@ image:https://travis-ci.org/kiegroup/optaweb-vehicle-routing.svg?branch=master[
 
 Web application for solving the https://www.optaplanner.org/learn/useCases/vehicleRoutingProblem.html[Vehicle Routing Problem]
 using https://www.optaplanner.org/[OptaPlanner].
+
+== Running the application
+
+First, clone this repository and `cd` into it.
+
+Frontend and backend are built and run independently. Open two terminal windows.
+
+== Run frontend
+
+. Install `npm`.
++
+```shell
+dnf install npm
+```
+
+. Install dependencies.
++
+```shell
```

Comment:
In the optaplanner docs, we always use this ascciidoc syntax for pieces of code:

```
[source,xml]
----
    <dependency>
      <groupId>org.optaplanner</groupId>
      <artifactId>optaplanner-benchmark</artifactId>
    </dependency>
----
```

We'll probably want to keep that consistent?

Reply:
+1, I didn't realize I'm using Markdown syntax.